tests/unit/Flash/SessionCest.php

```diff
@@ -35,8 +35,23 @@ public function _before(UnitTester $I)
         $this->newDi();
         $this->setDiEscaper();
         $this->setDiSessionFiles();
+
+        if (PHP_SESSION_ACTIVE !== session_status()) {
+            @session_start();
+        }
+
+        if (!isset($_SESSION)) {
+            $_SESSION = [];
+        }
     }
 
+    public function _after(UnitTester $I)
+    {
+        @session_destroy();
+    }
+
+
+
```

Comment:
Can these tests be run without suppressing the `session_start` and `session_destroy`? I really don't like the `@` because it might hide issues later on.